backend/python/app/models/__init__.py

```diff
@@ -1,7 +1,9 @@
+import os
+
 from flask_sqlalchemy import SQLAlchemy
 
 db = SQLAlchemy()
-erase_db_and_sync = False
+erase_db_and_sync = os.getenv("ERASE_DB_AND_SYNC", "False") == "True"
```

Comment:
added this so we can control this in heroku deployments using an environment variable but I can take it out if it's not necessary